org_status/org_status.py

```diff
@@ -137,4 +169,14 @@ def main():
 
         org_host = Host(token, org, verbose=args.verbose)
         org_status = aggregate_org_status(org_host, threads=args.threads)
+
+        if args.export_orgs:
+            export_data = encode_repo_list(org_status,
```

Comment:
You don't need to have this style of multiline parameters here, having them in a single line is not violating the guidelines.